torchtitan/parallelisms/parallelize_llama.py

```diff
@@ -38,8 +38,30 @@ def torch_spmd_parallelize(
     parallel_dims: ParallelDims,
     job_config: JobConfig,
 ):
+    # ensure full graph compile on non-llama models
+    torch._dynamo.config.capture_scalar_outputs = True
+    torch._dynamo.config.capture_dynamic_output_shape_ops = True
+
+    # simple fsdp configs
+    torch._inductor.config.simplefsdp.bucket_mode = "greedy"
     torch._inductor.config.simplefsdp.enable_reorder = True
-    torch._inductor.config.simplefsdp.enable_bucket = True
+    torch._inductor.config.simplefsdp.ag_comm_time_multiplier = 5
+    torch._inductor.config.simplefsdp.rs_comm_time_multiplier = 5
+    torch._inductor.config.simplefsdp.memory_constraint = 1000
```

Comment:
320 and 1000, which works? if 1000, then set it in the pytorch as well

Reply:
updated in pytorch